tests/Signals_perftest.m

```diff
@@ -1,90 +1,191 @@
 classdef Signals_perftest < matlab.perftest.TestCase
+% Runs performance tests for Signals
+%
+% Performs benchmarking for various operations on signals: 
+% nops, assignment ops, logical ops, scalar arithmetic ops, listener ops,
+% and signal methods: `map`, `scan`, `identity`, `subscriptable`, etc...
+% (via `test_mainOps` and `test_signalsOps` methods)
+%
+% And performs benchmarking for propogations through each signal in 
+% networks of various widths and depths (30 to 1000 nodes spread across 2
+% to 20 layers) (via `test_networkOps` method) (*Note: for reference, the 
+% `docs\examples\advancedChoiceWorld` exp def has 338 nodes over 10 layers)
+%
+% In this class, the 'Net' property is a Signals network, in which the 
+% 'A' & 'B' properties are treated as "input layer" signals from which all 
+% other signals are defined.
+%   
+% Example - display the mean measured time for running each test:
+%   results = runperf('Signals_perftest2.m');
+%   fullTableResults = vertcat(results.Samples);
+%   meanTimeByTestTable = varfun(@mean, fullTableResults,...
+%     'InputVariables', 'MeasuredTime', 'GroupingVariables', 'Name')
   
   properties
-    net
-    A
-    B
-    C
-    func
-    verify = false
-    results
-    reps = 2000
+    Net % Signals network ('sig.Net' object)
+    A % origin signal ('sig.node.OriginSignal' object)
+    B % origin signal ('sig.node.OriginSignal' object)
+    Reps = 1000 % number of test repetitions 
```

Comment:
I may be misremembering but I thought I had changed the reps to 2000 because some of the measurements were unreliable at 1000 reps (throwing a warning).

Reply:
1000 was working fine for me

Reply:
Fair enough